evalml/tests/component_tests/test_components.py

```diff
@@ -1628,3 +1646,33 @@ def test_estimator_fit_respects_custom_indices(
     estimator.fit(X, y)
     pd.testing.assert_index_equal(X.index, X_original_index)
     pd.testing.assert_index_equal(y.index, y_original_index)
+
+
+def test_component_parameters_modifies_feature_target():
```

Comment:
I'm not sure I understand the name of this test

Reply:
Relatable, copy pasta failure :)